### ISSUE TITLE
mkv container is missing metadata

### ISSUE THREAD

**Issue by buxel** [paulpacifico/shutter-encoder#184]
Hi,

I'm trying to transcode the videos i took with my phone for archiving. Since the iOS camera filenames do not give away the creation date, I tried to "preserve metadata" to keep a timestamp when the video was taken.

As you can see, only the "mp4" container has an "Media Created" proeperty after transcoding.
![image](https://github.com/paulpacifico/shutter-encoder/assets/331779/5b0e02c0-7848-4bb7-a874-6834cf50f013)

![image](https://github.com/paulpacifico/shutter-encoder/assets/331779/d8802acd-4ad4-4bbd-979b-1996c3099d10)

I have observed the "Media created" is kept, at least for .mp4 containers. 
Don't you think that "Date created" should also be  set according to source, if "preserve metadata" is selected?

Thank you for this lovely tool!

**Comment by paulpacifico** [comment]
Hi,

Unfortunately the .mkv metadata are handled by ffmpeg (the backend tool of SE) I can't do much for that but I'm updating it regularly.

Paul.

**Comment by buxel** [comment]
While searching for this issue, I found this: https://video.stackexchange.com/a/26076

> In my transcoder project, a lot of camera makers like to inject custom meta keys in the MP4/MOV container, and I want to keep them in the transcoded MP4/MOV files. After a lot of head scratching, FFmpeg does seem to have a switch for this purpose:

> -movflags use_metadata_tags

> Note that this has to go after the input file, as for example in:

> ffmpeg -i $input_file -movflags use_metadata_tags -crf 22 $output_file


according to a comment there, it should also resolve another issue I have with incorrect orients after changing scale of a video.

is SE already using this parameter?

**Comment by paulpacifico** [comment]
Yes SE is using `-movflags` argument but it's only working when you're using .mov files :-/

**Comment by buxel** [comment]
My source videos are all .MOV

**Comment by paulpacifico** [comment]
Yes but I speak about the output extension not input.

**Comment by buxel** [comment]
I see. Too bad, i was really hoping to transcode my phone videos to AV1 for logn term storage. Loosing the date information sadly is a no-go for this kind of content.

Thank you for clarifying!
